extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -5,9 +5,26 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		$wg = F::app()->wg;
 		$title = $wg->Title->getPrefixedDBkey();
 
-		if ( isset( $wg->articleVideoFeaturedVideos[$title] ) ) {
-			\Wikia::addAssetsToOutput( 'article_video_scss' );
-			\Wikia::addAssetsToOutput( 'article_video_js' );
+		if ( isset( $wg->articleVideoRelatedVideos ) ) {
+			$relatedVideo = ArticleVideoController::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title );
+		}
+
+		if ( ( $wg->enableArticleFeaturedVideo &&
+		       isset( $wg->articleVideoFeaturedVideos[$title] ) ) ||
```

Comment:
This is hard to read, please stick to the [guidelines](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#conditional-logic).